osquery/tables/system/windows/authenticode.cpp

```diff
@@ -242,12 +242,8 @@ Status getOriginalProgramName(SignatureInformation& signature_info,
     return Status(1, "Failed to access the publisher information");
   }
 
-  std::vector<std::uint8_t> publisher_info_blob_buffer;
-  publisher_info_blob_buffer.resize(static_cast<size_t>(publisher_info_size));
-  if (publisher_info_blob_buffer.size() !=
-      static_cast<size_t>(publisher_info_size)) {
-    return Status(1, "Memory allocation failure");
-  }
+  std::vector<std::uint8_t> publisher_info_blob_buffer(
+      static_cast<size_t>(publisher_info_size));
```

Comment:
Here and further is better to use `std::` prefix for type `size_t` [ref](https://en.cppreference.com/w/cpp/types/size_t)